front/app/containers/UsersEditPage/ProfileDeletion.tsx

```diff
@@ -71,9 +71,9 @@ class ProfileDeletion extends PureComponent<Props & InjectedIntlProps, State> {
             </Button>
           </Row>
         </FormSection>
-        <LoadableModal opened={dialogOpened} close={this.onCloseDialog}>
+        <Modal opened={dialogOpened} close={this.onCloseDialog}>
```

Comment:
Same here, the deletion dialog modal is ~150 lines of simple components. Not sure it needs to be lazily loaded vs. loading as part of the main component.